### ISSUE TITLE
Build TestNet Kibana Dashboards

### ISSUE THREAD

**Issue by yourbuddyconner** [CodaProtocol/coda-automation#12]
It would be useful to have live dashboards detailing some of the more complex interactions between components of a testnet. 

Dashboards we Should Have: 
- Network View (High-level overview of cluster state/health) 
- Node View (Lower-level breakdown of all of the processes/data structures running on a single node) 

Metrics That Would be Interesting: 
- Use baselines calculated for things like log volume from control test-nets and using them to compare to the currently running testnet

**Comment by yourbuddyconner** [comment]
Another thing to note is that it would be preferable to have automation in place to import these dashboards to any Kibana install. That way we won't get too attached to any particular ElasticSearch/Kibana instance. 

I see these dashboard JSON definitions being stored either in this repo or another companion repo for the purpose.
